src/plugins/gui/providers/paybisProvider.ts

```diff
@@ -584,8 +585,49 @@ export const paybisProvider: FiatProviderFactory = {
             const promoCodeParam = promoCode != null ? `&promoCode=${promoCode}` : ''
 
             if (direction === 'buy') {
+              const successReturnURL = encodeURIComponent('https://deep.edge.app/fiatprovider/buy/paybis?transactionStatus=success')
+              const failureReturnURL = encodeURIComponent('https://deep.edge.app/fiatprovider/buy/paybis?transactionStatus=fail')
               await showUi.openExternalWebView({
-                url: `${widgetUrl}?requestId=${requestId}${ott}${promoCodeParam}`
+                url: `${widgetUrl}?requestId=${requestId}${ott}${promoCodeParam}&successReturnURL=${successReturnURL}&failureReturnURL=${failureReturnURL}`,
+                providerId,
+                deeplinkHandler: async link => {
+                  const { query, uri } = link
+                  console.log('Paybis WebView launch buy success: ' + uri)
+                  const { transactionStatus } = query
+                  if (transactionStatus === 'success') {
+                    await showUi.trackConversion('Buy_Success', {
+                      conversionValues: {
+                        conversionType: 'buy',
+                        sourceFiatCurrencyCode: fiatCurrencyCode,
+                        sourceFiatAmount: fiatAmount,
+                        destAmount: new CryptoAmount({
+                          currencyConfig: coreWallet.currencyConfig,
+                          currencyCode: displayCurrencyCode,
+                          exchangeAmount: cryptoAmount
+                        }),
+                        fiatProviderId: providerId,
+                        orderId: requestId
+                      }
+                    })
+                    const message =
+                      sprintf(lstrings.fiat_plugin_buy_complete_message_s, cryptoAmount, displayCurrencyCode, fiatAmount, fiat, '1') +
+                      '\n\n' +
+                      sprintf(lstrings.fiat_plugin_buy_complete_message_2_hour_s, '1') +
+                      '\n\n' +
+                      lstrings.fiat_plugin_sell_complete_message_3
+                    await showUi.buttonModal({
+                      buttons: {
+                        ok: { label: lstrings.string_ok, type: 'primary' }
+                      },
+                      title: lstrings.fiat_plugin_buy_complete_title,
+                      message
+                    })
+                  } else if (transactionStatus === 'failure') {
+                    await showUi.showToast(lstrings.fiat_plugin_buy_failed_try_again, NOT_SUCCESS_TOAST_HIDE_MS)
+                  } else {
+                    await showUi.showError(new Error('Paybis: Invalid transactionStatus. This should never happen'))
```

Comment:
Maybe do it this way, so it's more debuggable: `showUi.showError(new Error('Paybis: Invalid transactionStatus "${transactionStatus}".'))`